### PR TITLE
Fix for large jsons

### PR DESCRIPTION
**Pull request by joemcbroom**
My formatting from prettier added `;` to a lot of lines so it looks like more changes than there actually are.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,46 +1,92 @@
...
+    const table_name = _.last(table_file.split('/')).split('.')[0];
+
+    console.log(`   Importing table ${table_file}...`);
+
+    if (table_list.includes(table_name)) {
+      await r.db(db_name).tableDrop(table_name).run(); //drop table, if exists, to clear data
+    }
+    await r.db(db_name).tableCreate(table_name).run(); // create table
+
+    const fileStream = fs.createReadStream(table_file);
```
Comment (joemcbroom) [inline index.js:33]: Here is the bulk of the change

**Comment by joemcbroom** [comment]
https://github.com/joemcbroom/rethinkdb-import/blob/94bdc41246a2a13e36e81de9e54a5c3e5349e4b7/index.js#L21-L53

here's the bulk of the new changes

File: index.js [hunk 2]
```diff
@@ -1,46 +1,92 @@
...
+      } else {
+        const chunked = chunkData(jsonData);
+        for (const chunk of chunked) {
+          await r.db(db_name).table(table_name).insert(chunk).run();
+        }
+      }
+      resolve();
+    });
+  });
+};
```
Comment (joemcbroom) [inline index.js:53]: here's the bulk of the changes

**Comment by GeoffreyPlitt** [comment]
This is excellent @joemcbroom, thank you!

**Pull request merged**